CONTRIBUTING.md

```diff
@@ -76,8 +76,9 @@ To be able to benefit from this feature, you will need to perform the following
 
 When building (`yarn build`) the project on the `main` branch (or another `release/*` branch), the remote cache will be downloaded.
 
-> [!IMPORTANT]
-> This feature is available for `@amadeus.com` email addresses, the Nx Cloud account should be created with an `@amadeus.com` email or it should be set as **main email** on GitHub if the GitHub account is used to register to Nx Cloud.
+> [!NOTE]
+> Only Otter Team member can write Remote Cache on Nx Cloud, the cache will use only to access build on pure release branches.
```

Comment:
```suggestion
> Only Otter team members can write remote cache on Nx Cloud, the cache will only be used to access builds on pure release branches.
```

Reply:
I don't know about the capitalization of `Remote Cache` (same thing for `Local Cache` below)...